libvips/resample/resize.c

```diff
@@ -134,13 +137,15 @@ vips_resize_build( VipsObject *object )
 	VipsResample *resample = VIPS_RESAMPLE( object );
 	VipsResize *resize = (VipsResize *) object;
 
-	VipsImage **t = (VipsImage **) vips_object_local_array( object, 7 );
+	VipsImage **t = (VipsImage **) vips_object_local_array( object, 5 );
 
 	VipsImage *in;
 	double hscale;
 	double vscale;
 	int int_hshrink;
 	int int_vshrink;
+	int target_width;
+	int target_height;
```

Comment:
These declarations could move inside the if() below.

Reply:
Resolved with commit 1282790529b1bf8cf5af26da431e3746b4be21c3.